api/defaults/service.go

```diff
@@ -79,5 +85,14 @@ func InterpolateService(origSpec *api.ServiceSpec) *api.ServiceSpec {
 		}
 	}
 
+	if spec.Rollback == nil {
+		spec.Rollback = Service.Rollback.Copy()
+	} else {
+		if spec.Rollback.Monitor == nil {
```

Comment:
I'm thinking we may add other fields that need to be handled here in the future. Right now there is only one, but putting it inside an `else` structures the code so that another one can be added along side it (even if it looks weird right now).